SwrveDemo/Podfile

```diff
@@ -0,0 +1,5 @@
+
+target 'SwrveDemoFramework' do
+pod 'SwrveSDK', '~> 4.4.0'
```

Comment:
It'll always be one version behind and we can't show new features on the demo then. How about having it local and commented that is the that line but pointing to the future 4.5.0 pod?